lib/src/component/ref_util.dart

```diff
@@ -46,8 +46,8 @@ import 'package:over_react/component_base.dart';
 ///     }
 ///
 /// Learn more: <https://reactjs.org/docs/refs-and-the-dom.html#creating-refs>.
-Ref<CurrentType> createRef<CurrentType>() {
-  return react_client.createRef<CurrentType>();
+Ref<T> createRef<T>() {
```

Comment:
Since this is all typed nicely in the `react.dart` layer now... can't we just do...

```dart
export 'package:react/react_client/react_interop.dart' show createRef;
```

Reply:
No; this is being re-exported so that we can use over_react syntax instead of react-dart syntax in the example